### ISSUE TITLE
"Cannot find module /es5/input/tex.js" error in Windows Node

### ISSUE THREAD

**Issue by curbengh** [mathjax/MathJax#2486]
### Issue Summary

I'm trying to integrate mathjax (tex => svg) into this node app (https://github.com/hexojs/hexo-math/pull/130).  Unit test ran fine in GitHub Actions' [Ubuntu](https://github.com/hexojs/hexo-math/runs/925692784?check_suite_focus=true) and [OSX](https://github.com/hexojs/hexo-math/runs/925692895?check_suite_focus=true) environments but failed in [Windows](https://github.com/hexojs/hexo-math/runs/925692842?check_suite_focus=true) with ` MathJax(input/tex): Cannot find module '/es5/input/tex.js'` error ([log](https://github.com/hexojs/hexo-math/pull/130/checks?check_run_id=925692842)).

[Affects](https://github.com/hexojs/hexo-math/pull/130/checks?check_run_id=925692842) Node 10, 12 & 14. Same issue in [Travis](https://github.com/hexojs/hexo-math/pull/130/checks?check_run_id=917004938).

### Steps to Reproduce:

[lib/mathjax.js](https://github.com/hexojs/hexo-math/blob/c9a817aa52324f65386cbad4133e377038aca292/lib/mathjax.js)

``` js
const { init } = require('mathjax');

module.exports = hexo => {
  return async function mathjaxFn(args, content) {
    if (this.mathjax === false) return content;

    const { options: globalCfg } = hexo.config.math.mathjax;
    const { mathjax: fmCfg } = this;
    const [jsonCfg] = args;
    const argsCfg = jsonCfg ? JSON.parse(jsonCfg) : false;
    let options = Object.assign({}, globalCfg);
    if (fmCfg) options = Object.assign({}, options, fmCfg);
    if (argsCfg) options = Object.assign({}, options, argsCfg);
    const { conversion, svg, tex } = options;

    const MathJax = await init({
      loader: {
        load: ['input/tex', 'output/svg']
      },
      tex,
      svg
    });

    const { startup, tex2svgPromise } = MathJax;
    const svgOut = await tex2svgPromise(content, conversion);

    return startup.adaptor.outerHTML(svgOut);
  };
};
```

Unit test [test/index.js](https://github.com/hexojs/hexo-math/blob/c9a817aa52324f65386cbad4133e377038aca292/test/index.js)

``` js
    describe('mathjax', () => {
      const m = require('../lib/mathjax')(hexo).bind({});
      const content = '\\frac{1}{x^2-1}';
      const displayFalse = '<mjx-container class="MathJax" jax="SVG">';
      const displayTrue = '<mjx-container class="MathJax" jax="SVG" display="true">';

      it('default', async () => {
        const output = await m(args, content);

        output.startsWith(displayFalse).should.eql(true);
      });
    });
```

### Technical details:

* MathJax Version: [3.0.5](https://github.com/hexojs/hexo-math/pull/130/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R41)
* OS: [Windows Server 2019](https://docs.github.com/en/actions/reference/virtual-environments-for-github-hosted-runners)
* Nodejs: [v10.21.0](https://github.com/hexojs/hexo-math/pull/130/checks?check_run_id=925692842), [v12.18.2](https://github.com/hexojs/hexo-math/pull/130/checks?check_run_id=925692856), [v14.5.0](https://github.com/hexojs/hexo-math/pull/130/checks?check_run_id=925692873)

### Supporting information:

https://github.com/hexojs/hexo-math/pull/130

Configuration [./index.js](https://github.com/hexojs/hexo-math/blob/c9a817aa52324f65386cbad4133e377038aca292/index.js)

``` js
  mathjax: {
    options: {
      // https://docs.mathjax.org/en/latest/web/typeset.html#conversion-options
      conversion: {
        display: false
      },
      // https://docs.mathjax.org/en/latest/options/input/tex.html
      tex: {},
      // https://docs.mathjax.org/en/latest/options/output/svg.html
      svg: {}
    }
  }
```

**Comment by dpvc** [comment]
Thanks for the report.  It looks like MathJax is having trouble locating its package directory in Windows.  I'm able to reproduce the problem, and will look into it further.  In the meantime, you can add

``` javascript
        paths: {mathjax: 'mathjax/es5'},
```

to the `loader` configuration passed to MathJax's `init()` function, and that should allow it to work on Windows (and everywhere else).  So you would use

``` javascript
    const MathJax = await init({
      loader: {
        paths: {mathjax: 'mathjax/es5'},
        load: ['input/tex', 'output/svg']
      },
      tex,
      svg
    });
```

See if that works for you.

**Comment by curbengh** [comment]
Thanks, WFM in [Actions](https://github.com/hexojs/hexo-math/pull/130/checks?check_run_id=934900886) & [Travis](https://travis-ci.com/github/hexojs/hexo-math/jobs/367301831)).

**Comment by dpvc** [comment]
It turns out that the copy of node's `path` module that webpack uses doesn't understand Windows paths that include backslashes, and that was the source of the problem, here.  I've made a PR (see link above) to resolve the issue.

**Comment by dpvc** [comment]
Fixed in v3.1 released today.
